Jenkinsfile

```diff
@@ -371,17 +371,14 @@ podTemplate(name: podName,
                                 // Send message org.centos.prod.ci.pipeline.allpackages.package.test.functional.complete on fedmsg
                                 pipelineUtils.sendMessageWithAudit(messageFields['topic'], messageFields['properties'], messageFields['content'], msgAuditFile, fedmsgRetryCount)
 
-                                try {
-                                    def testResults = pipelineUtils.parseTestLog("${WORKSPACE}/${currentStage}/logs/test.log")
-                                    testResults.each { test, result ->
-                                        if (result == 'FAILED') {
-                                            buildResult = 'UNSTABLE'
-                                        }
-                                        packagepipelineUtils.setMetricTag(env.fed_repo, test, result)
-                                    }
-                                } catch(err) {
-                                    buildResult = 'FAILED'
-                                }
+                                // parse the package tests log
```

Comment:
I don't think this will work. If the executeInContainer call to run the tests fails, it will never get to this section of code. And if test.log contains FAILED anywhere, the executeInContainer call fails by design.

Reply:
Makes sense. A fix was committed.